generators/docker-compose/index.js

```diff
@@ -53,12 +53,14 @@ module.exports = DockerComposeGenerator.extend({
             this.DOCKER_JHIPSTER_CONSOLE = constants.DOCKER_JHIPSTER_CONSOLE;
             this.DOCKER_JHIPSTER_ELASTICSEARCH = constants.DOCKER_JHIPSTER_ELASTICSEARCH;
             this.DOCKER_JHIPSTER_LOGSTASH = constants.DOCKER_JHIPSTER_LOGSTASH;
+            this.DOCKER_JHIPSTER_IMPORT_DASHBOARDS = constants.DOCKER_JHIPSTER_IMPORT_DASHBOARDS;
             this.DOCKER_JHIPSTER_ZIPKIN = constants.DOCKER_JHIPSTER_ZIPKIN;
             this.DOCKER_CONSUL = constants.DOCKER_CONSUL;
             this.DOCKER_CONSUL_CONFIG_LOADER = constants.DOCKER_CONSUL_CONFIG_LOADER;
             this.DOCKER_PROMETHEUS = constants.DOCKER_PROMETHEUS;
             this.DOCKER_PROMETHEUS_ALERTMANAGER = constants.DOCKER_PROMETHEUS_ALERTMANAGER;
             this.DOCKER_GRAFANA = constants.DOCKER_GRAFANA;
+            this.DOCKER_COMPOSE_VERSION = constants.DOCKER_COMPOSE_VERSION;
```

Comment:
The constant name is bit misleading. I thought it refers to the `docker-compose` executable version